src/ussd_app.js

```diff
@@ -230,16 +344,238 @@ go.app = function() {
                     " looking for?",
                 ].join("")),
                 choices: [
-                    new Choice("state_end", $("Nearest Clinic")),
-                    new Choice("state_end", $("MMC Clinic")),
-                    new Choice("state_end", $("HCT Clinic")),
+                    new Choice("nearest", $("Nearest Clinic")),
```

Comment:
@imsickofmaps Not quite sure what to do with this option as the original clinicfinder code only provided options for MMC and HCT... Does "Nearest Clinic" mean any type of clinic, whichever is closest..?  If I understood the code correctly, choosing the MMC and HCT options do lead to finding the nearest clinic of that type